css/src/components/layout.scss

```diff
@@ -207,3 +222,24 @@ $three-quarters-widescreen: math.div($breakpoint-widescreen, 4) * 3;
 		}
 	}
 }
+
+.layout.layout-constrained {
```

Comment:
On mobile and tablet, the footer seems to bleed into the content body  

![image](https://github.com/user-attachments/assets/713ac283-f2f6-4b7b-a9ab-fc0439b707b5)

![image](https://github.com/user-attachments/assets/3fd0612e-800c-45f0-b5f1-e6af470ac7e8)

Reply:
I've updated the solution to fix this issue. Still updating documentation though, since the holy grail layout isn't compatible with height constrains (due to its two rows in the middle of the page on tablet).